src/src/beneficiaries/beneficiaries.service.ts

```diff
@@ -652,15 +710,8 @@ return resp.status(404).send({
     };
 
     const response = await this.hasuraServiceFromServices.getData(data);
-
     let result = response?.data?.users_by_pk;
-    if (result?.beneficiaries && result?.beneficiaries[0]) {
-      result.beneficiaries = result.beneficiaries[0];
-    } else {
-      result = { ...result, beneficiaries: {} };
-    }
     let mappedResponse = result;
-
     return {
```

Comment:
send response like this:
return response.status(200).json({
                        success: true,
                        message: "User data fetched successfully!",
                        data: mappedResponse
                    });

Reply:
updated